src/index.html

```diff
@@ -1,14 +1,25 @@
 <!DOCTYPE html>
 <html>
   <body>
-    <div class="diamond_wordshift" style="display: flex; gap: 28em; margin-top: -70px; margin-left: -35px; justify-content: center;">
-      <div id="diamondplot"><svg xmlns="http://www.w3.org/2000/svg"></svg></div>
-      <div id="wordshift" style="margin-top: 110px;"><svg xmlns="http://www.w3.org/2000/svg"></svg></div>
+    <div style="display:flex; align-items:center; gap: 10em; margin-left: -75px; justify-content: center; width: 100%; text-align: center; font-size: 22px; ">
+      <div> Girls 1890</div> 
+      <div> Girls 1885</div>  
+    </div>
```

Comment:
Thanks for pointing out. I agree we should this. Does JS have a way to set up optional args like in Python? I'd want the user to optionally be able to pass them (otherwise it's so many required args)

Reply:
This is what it looks like for me. I think currently leaving the system names may be confusing--put placeholder?
<img width="1298" alt="Screenshot 2024-10-30 at 12 11 03 PM" src="https://github.com/user-attachments/assets/026c2c97-a488-4bd8-ba88-8eab88b233b9">

Reply:
I think the balance-plot and wordshift-plot overlap at the bottom can be okay. Idk if that's an easy fix. It looks like some margins may bumped it a bit.

Reply:
Last, delete index2.html? It was meant to be an equivalent with CSS pulled out--better just to keep 1 bc this one is changing.

Reply:
Shoot, I just noticed this is producing a 2nd blank page in the pdf now.

Reply:
In order:
- There are a bunch of ways we could do that in JS.
- About the overlapping there is still some improvement that could be made in the assemblage of plots. But i consider that to be a minor issues that can be fixed after the PR.
- About the 2nd blank page: i was wondering about that too. I wasn't if the 2nd blank page was there before or not. Not sure why it happens.

Reply:
> This is what it looks like for me. I think currently leaving the system names may be confusing--put placeholder? <img alt="Screenshot 2024-10-30 at 12 11 03 PM" width="1298" src="https://private-user-images.githubusercontent.com/63805609/381637936-026c2c97-a488-4bd8-ba88-8eab88b233b9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzMDcxMzksIm5iZiI6MTczMDMwNjgzOSwicGF0aCI6Ii82MzgwNTYwOS8zODE2Mzc5MzYtMDI2YzJjOTctYTQ4OC00YmQ4LWJhODgtOGVhYjg4YjIzM2I5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMwVDE2NDcxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE0MjdiODc1NGI2NjY1YzM3N2NmNGQwN2MyNjRhOWVjZjc0NzdmYjFiZWI2NTdkYjZiYWMwMTUyYjc3ZmIxOGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.1tcj8dibwQsKlRJUOClz5s2M--_XLFPAnpZdW6l59f4">

ohh, thats wrong. Not sure why. What is your alpha?